streams/src/main/java/org/apache/kafka/streams/state/internals/metrics/Sensors.java

```diff
@@ -25,7 +31,8 @@
 import static org.apache.kafka.streams.processor.internals.metrics.StreamsMetricsImpl.addInvocationRateAndCount;
 
 public final class Sensors {
-    
+    private static final String PROCESSOR_NODE_METRICS_GROUP = "stream-processor-node-metrics";
```

Comment:
Does it make sense to use the same group name as above?

Reply:
This string can effectively never change (it would mean we've renamed all our processor metrics), so I don't think there's any risk from the duplication.

Reply:
It's not about any risk, but a semantic question. Is it correct to use the same name (ie, right hand side) for both? And if yes, why so we want this?